packages/swagger/src/generators/OperationGenerator.ts

```diff
@@ -3,11 +3,12 @@ import { pascalCase } from 'change-case'
 import type { FileManager, File } from '@kubb/core'
 import { Generator } from '@kubb/core'
 
+import { isReference } from '../utils/isReference'
```

Comment:
Did not mean to place this import somewhere else, must have been the prettier/eslint setup